docs/lua-records/reference/netmask.rst

```diff
@@ -42,10 +42,22 @@ The :class:`Netmask` class represents an IP netmask.
       Returns a :class:`ComboAddress` representing the network (truncating according to the mask).
 
   .. method:: Netmask:isIpv4() -> bool
+  .. deprecated:: v4.x.x
```

Comment:
I believe we need an empty line before `deprecated` and `versionadded`, see #8437. Same thing in pdns/recursordist/docs/lua-scripting/netmask.rst, unfortunately :'(

Reply:
fixed.